### ISSUE TITLE
Querying wmslayer does not seem to work in release build

### ISSUE THREAD

**Issue by kalessine** [mapserver/mapserver#5900]
Querying a wms layer that points to a wms service does not seem to work in the release build (rel-7-4-2) instead we get no text at all returned by getFeatureInfo on the parent layer...
this is a snipper of our parent layer information;
 LAYER
    CONNECTION "http://localhost/cgi-bin/mapserv?map=/maps/zzz.map&"
    CONNECTIONTYPE WMS
    METADATA
      "wms_srs" "EPSG:900913 EPSG:4326"
      "wms_extent"      "13048989.6381759 -4108382.56980896 13049321.6706854 -4107853.60807286"
      "wms_layer_group" "/2019-09-27/09:00  -  test (ID: 15044)"
      "wms_name"        "layer56_simulation_15044"
      "wms_feature_info_mime_type"      "text/html"
      "wms_format"      "image/png"
      "wms_server_version"      "1.1.1"
      "wms_title"       "zzzz (Default)"
    END # METADATA
    NAME "layer56_simulation_15044"
    PROJECTION
      "init=epsg:900913"
    END # PROJECTION

Thankyou!

**Comment by jratike80** [comment]
By the documentation https://www.mapserver.org/ogc/wms_client.html

> GetFeatureInfo is not fully supported since the output of getFeatureInfo is left to the discretion of the remote server. A method layer.getWMSFeatureInfoURL() has been added to MapScript for applications that want to access featureInfo results and handle them directly.

I wonder if it is supported at all. At least `"wms_feature_info_mime_type" "text/html"` probably has no effect because it is not mentioned in the WMS client documentation.

You created a bug report about this instead of asking a question from the mapserver-users mailing list, does it mean that GetFeatureInfo has worked for you before with an older Mapserver version?

**Comment by kalessine** [comment]
yes, this has worked in the version of mapserver that is in the ubuntugis stable ppa for ubuntu 14.04
mapserver | 7.0.7-1~trusty3 | Angelos Tzotsos (2019-05-16)
-- | -- | --
when run on ubuntu 14.04
I have tried recompiling the release 7-4-2 with the flags  -DWITH_WMSCLIENT=1 -DUSE_GDAL=1 -DUSE_PROJ=1
but none of my compiles will answer a GetFeatureInfo request :(
when running on ubuntu 18.04, or 16.04
I believe it has something to do with libcurl4 breaking backwards compatibility with libcurl4
but i'm only guessing, i tried to get libcurl3 loaded along with mapserver by adding an environment variable according to this guide:
https://dev.to/jake/using-libcurl3-and-libcurl4-on-ubuntu-1804-bionic-184g
to set the environment variable LD_PRELOAD=/usr/lib/libcurl.so.3 in the default.conf file for nginx where mapserver is called, but i still didn't get output from GetFeatureInfo  (but all the other calls my app uses to the parent wms layer worked)
Thanks!
